ui/imports/Utils.qml

```diff
@@ -139,4 +139,20 @@ QtObject {
             }
         }
     }
+
+    function getEmojiCodepoint(iconCodePoint) {
+        // Split the codepoint to get all the parts and then encode them from hex to utf8
+        const splitCodePoint = iconCodePoint.split('-')
+        let codePointParts = []
+        splitCodePoint.forEach(function (codePoint) {
+            codePointParts.push(`0x${codePoint}`)
+        })
+        return String.fromCodePoint(...codePointParts);
+    }
```

Comment:
What do you think about moving this to `/ui/imports/Emoji.qml` as that file already has all the js Emoji functions in it, including the reverse of this function?

Reply:
In saying that, this function is different from `Emoji.fromCodePoint`, correct? For consistency, maybe we should call it `toCodePoint` instead, and once moved to `Emoji.qml`, would be used as `Emoji.toCodePoint`.

Reply:
I moved it to Emoji like you said. However, I didn't rename it as `toCodePoint`, since Twemoji already has a `toCodePoint` function and it doesn't do the same thing (I tried)